sampleproject/codelite/build_settings.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That file is generated by Codelite when opened interactively.

According to https://forums.codelite.org/viewtopic.php?f=11&t=4571
we cannot create it from command line, so versioned :/.